src/services/auth-http.ts

```diff
@@ -15,3 +18,21 @@ export const cloudSettings: CloudSettings = {
     'app_id': 'APP_ID'
   }
 };
+
+export function handleError(error: Response | any, caught: any) {
```

Comment:
These two abstractions are great, but why is the service called `auth-http`? Might be better named `http-helpers` or something.